src/main/java/de/tum/cit/aet/artemis/lecture/web/LectureUnitResource.java

```diff
@@ -200,11 +200,26 @@ public ResponseEntity<LectureUnitForLearningPathNodeDetailsDTO> getLectureUnitFo
      */
     @GetMapping("lecture-units/{lectureUnitId}")
     @EnforceAtLeastStudent
-    public ResponseEntity<LectureUnit> getLectureUnitById(@PathVariable @Valid Long lectureUnitId) {
+    public ResponseEntity<LectureUnit> getLectureUnitById(@PathVariable @Valid long lectureUnitId) {
         log.debug("REST request to get lecture unit with id: {}", lectureUnitId);
         var lectureUnit = lectureUnitRepository.findByIdWithCompletedUsersElseThrow(lectureUnitId);
         authorizationCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.STUDENT, lectureUnit.getLecture().getCourse(), null);
         lectureUnit.setCompleted(lectureUnit.isCompletedFor(userRepository.getUser()));
         return ResponseEntity.ok(lectureUnit);
     }
+
+    /**
+     * POST lectures/{lectureId}/lecture-units/{lectureUnitId}/ingest
+     * This endpooint is for starting the ingestion of one lecture unit.
+     *
+     * @param lectureUnitId The Id of the Lecture Unit that is <going to be ingested
+     * @return the ResponseEntity with status 200 (OK) and a message success or null if the operation failed
+     */
+    @PostMapping("lectures/{lectureId}/lecture-units/{lectureUnitId}/ingest")
+    @EnforceAtLeastInstructor
+    public ResponseEntity<Void> ingestLectureUnit(@PathVariable long lectureUnitId) {
+        LectureUnit lectureUnit = this.lectureUnitRepository.findByIdWithCompetenciesAndSlidesElseThrow(lectureUnitId);
```

Comment:
do you really need competencies and slides? or could you also just fetch the lecture unit?

Reply:
As far as I can tell from looking at the service methods, you can simply fetch the lecture unit

Reply:
The slides are needed but the competencies not. Thus I changed the call to fetch only the slies with the lecture unit.

Reply:
We need the slides to send them to pyris.

Reply:
Yes